.vscode/settings.json

```diff
@@ -8,6 +8,9 @@
       "editor.defaultFormatter": "esbenp.prettier-vscode"
     },
     "prettier.enable": true,
+    "rust-analyzer.rustfmt.extraArgs": [
+      "+nightly"
+    ],
```

Comment:
Additional to https://github.com/facebook/relay/pull/3959, this helps contributors use the same setup as [the github rust lint action](https://github.com/facebook/relay/blob/068608c7f799a7558a3b4abd666f06851f1285b8/.github/workflows/ci.yml#L109)